README.md

```diff
@@ -25,136 +25,33 @@
-- **Ninja:** We strongly encourage to work with `ninja` (instead of `make`) for faster compile times.
-
-## External dependencies
-
-4C heavily relies on the [Trilinos project](https://trilinos.github.io).
-
-Some further third-party libraries (TPLs) are mandatory, e.g.
-
-- CMake (minimum version: 3.30)
-- ParMETIS (recommended version: 4.0.3)
-- SuiteSparse (recommended version: 5.4.0)
-- SuperLUDist (recommended version: 7.2)
-- Qhull (recommended version: 2012.1)
-- CLN (recommended version: 1.3.4)
-
-and some are optional, e.g.
-
-- FFTW
-- [ArborX](https://github.com/arborx/ArborX)
-- [MIRCO](https://github.com/imcs-compsim/MIRCO/)
-
-Maybe, a pre-compiled version of Trilinos and set of TPLs is available at your institute.
-Look into the CMake presets in `presets/` or ask your colleagues for further information.
-
-Some helper scripts to install these TPLs can be found in `dependencies/`.
-
-Additional information can be found in
-the [user documentation](https://4c-multiphysics.github.io/4C/readthedocs/4Csetup.html#external-dependencies).
-
-## Getting up and running
-
-### Create python virtual environment (required for code development in 4C)
-
-For testing and active development, you need to create a python virtual environment once.
-In the source directory,
-execute:
-
-```
-./utilities/set_up_dev_env.sh
-```
-
-### Configure and Build
-
-4C enforces an out-of-source build, i.e., the build directory may not be the top-level source directory.
-Create a build directory `<path/to/build/dir>` of your choosing.
-
-Navigate into the build directory and run
-
-```bash
-cmake --preset=<name-of-preset> <path/to/source/dir>
-```
-
-A preset name needs to be passed to cmake via the command line argument `--preset`.
-Use `cmake <path/to/source/dir> --list-presets` to get a list of all available presets.
-Define your own presets in a `CMakeUserPresets.json` file.
-
-More information about CMake presets can be found in the [CMake documentation](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html).
-
-#### Build
-
-To build the code on `<numProcs>` processes, run
-
-```bash
-ninja -j <numProcs> full
+To quickly run 4C, you can use our docker image, where 4C comes pre-compiled.
 ```
-
-#### Run the Tests
-
-To verify that the build was successful, run the minimal set of tests via
-
-```bash
-ctest -L minimal
-```
-
-or all tests via
-
-```bash
-ctest
-```
-
-You can use the option `-j <num_threads>` to specify the number of threads to be used for parallel execution.
-
-### Prepare and Run Simulations
-
-After successfully building 4C, the executable `4C` is located in your build directory `<path/to/build/dir>/`.
-It needs to be invoked together with an input (`.dat`) file via
-
-```bash
-<path/to/build/dir>/4C <jobName>.dat <outputName>
+docker run --interactive --tty ghcr.io/4c-multiphysics/4c:main
+/home/user/4C/build/4C ../tests/input_files/<some-input-file>.4C.yaml output_name
```

Comment:
I'm not too happy about docker. It makes it very easy to use 4C. However, it is pretty cumbersome to get the simulation results out of the docker container.